lib/torque/renderer/point.js

```diff
@@ -313,6 +313,28 @@ var Filters = require('./torque_filters');
       return positions;
     },
 
+    /**
+     * returns an array with all the values for the active pixels
+     * @tile tile object
+     * @step integer with the step
+     * @values (optional) an array where the values will be placed
+     */
+    getValues: function(tile, step, values) {
```

Comment:
If you don't think there'll be confusion with the method naming, I've no objections.